<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>571</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>571</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pruebas Vehículo Eléctrico</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>530</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>530</height>
    </size>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>781</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="tab_frenado">
        <attribute name="title">
         <string>Prueba de frenado</string>
        </attribute>
        <widget class="QLabel" name="label_fre_fase">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>50</y>
           <width>311</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>Eficiencia en frío</string>
         </property>
        </widget>
        <widget class="PlotWidget" name="widget_fre_plot" native="true">
         <property name="geometry">
          <rect>
           <x>340</x>
           <y>10</y>
           <width>421</width>
           <height>421</height>
          </rect>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_fre_titulo">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>317</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Prueba de frenado</string>
         </property>
        </widget>
        <widget class="Line" name="line_2">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>10</y>
           <width>16</width>
           <height>421</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_fre_Ti">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>250</y>
           <width>161</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>Empty</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_fre_vx">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>90</y>
           <width>161</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>Empty</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_fre_fp">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>170</y>
           <width>161</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>Empty</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_fre_4">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>210</y>
           <width>131</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>df</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_fre_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>130</y>
           <width>131</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>ax (m/s^2)</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_fre_df_Td">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>210</y>
           <width>161</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>Empty</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_fre_5">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>250</y>
           <width>131</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>Ti (°C)</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_fre_ax">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>130</y>
           <width>161</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>Empty</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_fre_3">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>170</y>
           <width>131</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>fp (N)</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_fre_1">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>90</y>
           <width>131</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>vx (km/h)</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_fre_t_dr">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>290</y>
           <width>161</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>Empty</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_fre_6">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>290</y>
           <width>131</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>t (s)</string>
         </property>
        </widget>
        <widget class="QPushButton" name="button_fre_fase">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>380</y>
           <width>161</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Siguiente fase</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_estabilidad">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <attribute name="title">
         <string>Prueba de estabilidad</string>
        </attribute>
        <widget class="PlotWidget" name="widget_est_plot" native="true">
         <property name="geometry">
          <rect>
           <x>340</x>
           <y>10</y>
           <width>421</width>
           <height>421</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_33">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>317</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Prueba de estabilidad</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_34">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>50</y>
           <width>131</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>vx (km/h)</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_est_delta">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>130</y>
           <width>161</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>Empty</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_est_v">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>90</y>
           <width>161</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>Empty</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_36">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>130</y>
           <width>131</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>δ (°)</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_est_ay">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>50</y>
           <width>161</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>Empty</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_35">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>90</y>
           <width>131</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>ay (m/s^2)</string>
         </property>
        </widget>
        <widget class="Line" name="line">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>10</y>
           <width>16</width>
           <height>421</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_fre_b_5">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>170</y>
           <width>131</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>β* (°)</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_fre_b_6">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>210</y>
           <width>131</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>φ* (°)</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_fre_beta">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>170</y>
           <width>161</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>Empty</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_fre_phi">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>210</y>
           <width>161</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>Empty</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Prueba de vibraciones</string>
        </attribute>
        <widget class="QLabel" name="label_57">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>50</y>
           <width>131</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>vx (km/h)</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_est_delta_3">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>130</y>
           <width>161</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>Empty</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_est_v_5">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>90</y>
           <width>161</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>Empty</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_58">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>130</y>
           <width>131</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>δ (°)</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_est_ay_5">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>50</y>
           <width>161</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>Empty</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_59">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>317</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Prueba de vibraciones</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_60">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>90</y>
           <width>131</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>ay (m/s^2)</string>
         </property>
        </widget>
        <widget class="Line" name="line_9">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>10</y>
           <width>16</width>
           <height>421</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="PlotWidget" name="widget_est_plot_5" native="true">
         <property name="geometry">
          <rect>
           <x>340</x>
           <y>10</y>
           <width>421</width>
           <height>421</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QPushButton" name="startButton">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Iniciar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="countButton">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Detener</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMen">
    <property name="title">
     <string>Menú</string>
    </property>
   </widget>
   <addaction name="menuMen"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
